mantle/platform/api/gcloud/compute.go

```diff
@@ -33,8 +34,56 @@ func (a *API) vmname() string {
 	return fmt.Sprintf("%s-%x", a.options.BaseName, b)
 }
 
+// ["local-ssd:interface=NVME,size=375"]
+func ParseDiskSpec(spec string, zone string) (*compute.AttachedDisk, error) {
+	split := strings.Split(spec, ":")
+	var disktype string
+	var diskinterface string
+	var disksize string
+	var size int64
+	var err error
+	disk_ps := "PERSISTENT"
```

Comment:
By default, persistent disks created like this are still automatically cleaned up when the instance is terminated, correct?

Reply:
I don't think so actually, https://cloud.google.com/compute/docs/disks/persistent-disks mentions they can be moved between VMs.  Also https://cloud.google.com/compute/docs/disks#pdspecs
says
"Persistent disks are located independently from your virtual machine (VM) instance".

Reply:
Dug more into this. The `AutoDelete` flag below ensures that it's still lifecycled to the VM (see https://cloud.google.com/compute/docs/disks/modify-persistent-disk#updateautodelete). We actually [already use `PERSISTENT` today](https://github.com/coreos/coreos-assembler/blob/024ce76e47e01d9e59e2582709baf6792823c47d/mantle/platform/api/gcloud/compute.go#L77) in kola (and I see now @HuijingHei linked to this [in her comment](https://github.com/coreos/fedora-coreos-tracker/issues/1457#issuecomment-1549144654)) with auto-deletion on.

So resolving this! :)